material-overrides/main.html

```diff
@@ -74,6 +74,35 @@
   ></script>
 {% endblock %}
 
+{% block site_nav %}
```

Comment:
It looks like some lines were removed because now the left navigation isn't showing up. Nothing should have been removed, only should have added the feedback and actions html files.